pages.fr/common/mongo.md

```diff
@@ -0,0 +1,20 @@
+# mongo
+
+> Client shell pour MongoDB.
+> Plus d'informations: <https://docs.mongodb.com/manual/reference/program/mongo>.
+
+- Connecte à une base de données (database) :
+
+`mongo {{nom_de_la_base_de_données}}`
+
+- Connecte à une base de données (database) sur un hôte (host) distant et un port donné :
+
+`mongo --host {{hôte}} --port {{port}} {{nom_de_la_base_de_données}}`
+
+- Connecte à une base de données (database) avec un nom d'utilisateur (username): L'utilisateur sera invité à saisir son mot de passe :
```

Comment:
Is `;` used the same way in French as in English?
```suggestion
- Connecte à une base de données (database) avec un nom d'utilisateur (username); L'utilisateur sera invité à saisir son mot de passe :
```

Reply:
Yes in french a coma is possible in that case. i'll update